NewHaven/src/GameController.cpp

```diff
@@ -137,6 +136,37 @@ void GameController::playTurn(){
 // end game function
 void GameController::endGame(){
 // requires part4 and isnt part of what is needed for Part 2 driver
+scores winners = findWinner();
+// two cases one where ther eis only one winner, or a tie
+string announcement{""};
+if(winners.size() == 1)
+    announcement = "THE WINNER IS: ";
+
+else
+    announcement = "THE WINNERS ARE:\n";
```

Comment:
You don't need to assign anything to the string `announcement` when on declaration when you're not going to use the value assigned.